### PR TITLE
Update evaluate.py to remove policy flag and add log directory flag

### PR DESCRIPTION
**Pull request by JenishPatel99**
Addressing issue : #519 

Changes made
- Removed the policy flag from evaluate.py to prevent unnecessary policy input
- Added log-dir flag option to evaluate.py to request designated directory for new experiments
- Updated evaluate_test.py to conform to new log-dir flag

### CODE REVIEW

**Review by Gamenot — APPROVED** [review]

File: ultra/tests/test_train.py [hunk 1]
```diff
@@ -54,37 +54,38 @@ def test_locate_log_directory(self):
...
 
-    # def test_train_single_agent(self):
-    #     if os.path.exists("ultra/tests/logs"):
-    #         shutil.rmtree("ultra/tests/logs")
+    def test_train_single_agent(self):
+        if os.path.exists("ultra/tests/logs"):
+            shutil.rmtree("ultra/tests/logs")
 
-    #     os.system("pkill -9 ray")
+        os.system("pkill -9 ray")
```
[the remark this answers is not in the document]
Comment (JenishPatel99) [reply ultra/tests/test_train.py]: I was wondering if we are going to replace ray with another distributed solution? #549

Apart from killing the ray processes, I tried to ray.shutdown() prior to new ray executions (ray.init()). This eliminated the ray leakage. I felt like it was worth sharing

File: ultra/evaluate.py [hunk 2]
```diff
@@ -156,15 +159,9 @@ def evaluate(
...
-    parser.add_argument(
-        "--policy",
-        help="Policies available : [ppo, sac, ddpg, dqn, bdqn]",
-        type=str,
-        default="sac",
-    )
     parser.add_argument("--models", default="models/", help="Directory to saved models")
     parser.add_argument(
-        "--episodes", help="Number of training episodes", type=int, default=200
+        "--episodes", help="Number of training episodes", type=int, default=5
```
Comment (AlexLewandowski) [inline ultra/evaluate.py:164]: I think we can leave this the same for now. If not, we should update the README as well.

**Review by AlexLewandowski — APPROVED** [review]

File: ultra/tests/scenarios/task00/config.yaml [hunk 3]
```diff
@@ -28,4 +28,42 @@ levels:
...
+          specs: [[50kmh,low-density,0.21],[70kmh,low-density,0.20],[100kmh,low-density,0.20], #61%
+                  [50kmh,mid-density,0.11],[70kmh,mid-density,0.11],[100kmh,mid-density,0.11], #33%
+                  [50kmh,no-traffic,0.01],[70kmh,no-traffic,0.01],[100kmh,no-traffic,0.01], #3%
+                  [50kmh,high-density,0.01],[70kmh,high-density,0.01],[100kmh,high-density,0.01]] # 3%
+        3lane_c:
+          percent: 0.5
+          specs: [[50kmh,low-density,0.21],[70kmh,low-density,0.20],[100kmh,low-density,0.20], #61%
+                  [50kmh,mid-density,0.11],[70kmh,mid-density,0.11],[100kmh,mid-density,0.11], #33%
+                  [50kmh,no-traffic,0.01],[70kmh,no-traffic,0.01],[100kmh,no-traffic,0.01], #3%
+                  [50kmh,high-density,0.01],[70kmh,high-density,0.01],[100kmh,high-density,0.01]] # 3%
```
Comment (AlexLewandowski) [inline ultra/tests/scenarios/task00/config.yaml:69]: Is eval_test needed for the additional specs or something else?

Comment (JenishPatel99) [reply ultra/tests/scenarios/task00/config.yaml:69]: So yeah, what I was encountering when using simple scenarios (--task 00 --level easy) was that the evaluation process was really slow. I decided to add additional diverse scenarios, which then allowed to decreases the number of steps per episode. The eval test is merely used for test_evaluation.py

File: ultra/evaluate.py (hunk 2 above)
Comment (JenishPatel99) [reply ultra/evaluate.py:164]: oops, I forgot to revert it back to its original. value I will do it right now

**Pull request merged**